src/pages/Login/index.js

```diff
@@ -75,21 +80,51 @@ function Login(props) {
   }
 
   return (
-    <div className="ng-login">
-      <div className="logo" />
-      {loading ? (
-        <Loader />
-      ) : (
-        <GoogleLogin
-          clientId={process.env.REACT_APP_GOOGLE_CLIENT_ID}
-          buttonText="Continue with Google"
-          onSuccess={onSignIn}
-          onFailure={onGoogleLoginFail}
-          cookiePolicy={"single_host_origin"}
-          className="google-login"
-        />
-      )}
-    </div>
+    <>
+      <Container className={classes.merakiLogin} maxWidth="lg">
+        <Grid container spacing={2}>
+          <Grid item xs={12} ms={6} md={6}>
+            <Container maxWidth="sm">
+              <Typography
+                sx={{ pt: { xs: "none", md: 24 } }}
+                variant="h4"
+                align={isActive ? "center" : "left"}
```

Comment:
You probably don't want to use `align` as this will produce this obsolete attribute in the resulting HTML.  Also, as you seem to be using this type of aligning for a number of different elements, maybe it'd be useful to create a file with these styles.  For example:
```
const smAlignStyle =
  (smallAlign, notSmallAlign) =>
  ({ theme }) => ({
    [theme.breakpoints.down('sm')]: {
      textAlign: smallAlign,
    },
    [theme.breakpoints.up('sm')]: {
      textAlign: notSmallAlign,
    },
  });

export const centerSmRightNonSmAlignStyle = smAlignStyle('center', 'right');
export const centerSmLeftNonSmAlignStyle = smAlignStyle('center', 'left');
```

Then you could import these constants and have the following stylized component for use:
```
const LeftCenterAlignedTypography = styled(Typography)(centerSmLeftNonSmAlignStyle);
```

I also mention a similar suggestion in the Profile PR: https://github.com/navgurukul/bhanwari-devi/pull/249#discussion_r830564262.